### ISSUE TITLE
install.hs build does not build 'hie-8.6.3' correctly

### ISSUE THREAD

**Issue by alanz** [haskell/haskell-ide-engine#1157]
Running `hie-8.6.3` gives

```
2019-04-04 21:56:22.251873459 [ThreadId 4] - Run entered for HIE(hie-8.6.3) Version 0.7.0.0, Git revision bfbbc6c28b0980715547d7ddafb52057e2a985f5 (dirty) (2480 commits) x86_64 ghc-8.6.4
```

i.e. it is built with GHC 8.6.4

**Comment by fendor** [comment]
Should be fixed in head, cause is [#1131](https://github.com/haskell/haskell-ide-engine/issues/1131)

**Comment by alanz** [comment]
Yes it is, I was building the wrong branch.  Sorry.
